compiler/utils.go

```diff
@@ -643,3 +644,38 @@ func endsWithReturn(stmts []ast.Stmt) bool {
 func encodeIdent(name string) string {
 	return strings.Replace(url.QueryEscape(name), "%", "$", -1)
 }
+
+// formatJSTag returns a string of Javascript code appropriate for accessing
+// the property identified by jsTag. If the jsTag value can be safely encoded
+// in Javascript using the dot notation this is used, else we fallback to the
+// bracket notation. For more details see:
+//
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Property_Accessors
+//
+// Uses definition of an identifier from
+// https://developer.mozilla.org/en-US/docs/Glossary/Identifier
+func formatJSTag(jsTag string) string {
+	useDot := true
+
+RuneRange:
```

Comment:
Trivial style nitpick, maybe use `Outer` label. It'd be more consistent with:

https://github.com/gopherjs/gopherjs/blob/558a9132744c22476178edf3126fd35a9754f565/build/build.go#L162

(In general, `RuneRange` is not a label I see often, but `Outer` is pretty common.)

Reply:
Done